README.md

```diff
@@ -1,2 +1,124 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+```
+GET /accounts/{accountName}/campaigns/{campaignId}/content/design
+```
+
+Response:
+```
+{
+	"counters":{},
+	"body": {},
+	"schemaVersion": 6
+}
+```
+
+<br>
+
+## Thumbnails
+```
+GET /accounts/{accountName}/campaigns/{campaignId}/content/thumbnail
+```
+
+Response:
+<br>
+PNG/JPG file or redirect to image URI
+
+<br>
+
+## Save Campaign
+```
+PUT /accounts/{accountName}/campaigns/{campaignId}/content/
+```
+
+Response:
+```
+{
+	"counters":{},
+	"body": {},
+	"schemaVersion": 6,
+	"HTML": "",	// HTML
+	"thumbnailUrl": "" // URI Image
+}
+```
```

Comment:
In this case, the JSON is not the response, it is the request body.

(The usage of `. . .` is arguable)

```suggestion
\```
PUT /accounts/{accountName}/campaigns/{campaignId}/content/`

{
	"counters": { . . . },
	"body": { . . . },
	"schemaVersion": 6,
	"HTML": " . . . ",
	"thumbnailUrl": " . . . "
}
\```
```

Reply:
Fixed!